<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QWidget" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="et_timer_client">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_hex_receive_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>十六进制接收</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_timer_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>定时发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_bind_port_client">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_client">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>10</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_search_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_send_client">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search_client">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_char_receive_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字符接收</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定IP:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send_client">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_hex_send_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>十六进制发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_char_send_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字符发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_bind_ip_client">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_export_client">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose_file_client">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>定时时长：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_import_client">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导入数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_server_ip_client">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_server_port_client">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect_client">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_file_dir_client">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
